2_process/src/munge_nwis_SpC_data.R

```diff
@@ -1,72 +0,0 @@
-munge_daily_mean_SpC_cols <- function(x){
```

Comment:
Did you use `git mv` to rename a file? Git will track that as a rename instead of a deletion, and I think GitHub would also track this as a rename. Not important to change, just a tip for future

Reply:
I actually did delete this file `munge_nwis_SpC_data.R` because it was extraneous after I incorporated dynamic branching in the phase file and column name cleaning in the `get_daily_nwis_data.R` file. That's good to know about `git mv`!